src/components/app/ProfileName.css

```diff
@@ -17,6 +17,8 @@
 .profileNameButton {
   overflow: hidden;
   padding: 0 9px;
+  background: url(firefox-profiler-res/img/svg/edit-name-profiler.svg) no-repeat
```

Comment:
That's not the right way to insert the icon, you can look at other examples such as https://github.com/firefox-devtools/profiler/blob/9f69b30ae0e2408c191075da6b8c2b85112907c0/src/components/app/MenuButtons/index.css#L101-L103

Can you please try this?

Reply:
Yes sure, And again sorry for the inconvenience.